src/BannerWithLink/index.stories.tsx

```diff
@@ -8,7 +8,7 @@ import { grid, withGrid } from '../../.storybook/grid/withGrid';
 
 export default {
     component: 'BannerWithLink',
-    title: 'Banner/BannerWithLink',
+    title: 'WorkInProgress/Banner/BannerWithLink',
```

Comment:
@tjmw I think this is all it takes to move things to WIP (Just borrowing the approach from a previous [PR](https://github.com/guardian/braze-components/pull/284))

Reply:
Yeah, that looks right to me! 👍